Add doc for EnableESSwithSupportedPeripherals (#10877)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/passportforwork-csp.md

*Not supported on Windows Holographic and Windows Holographic for Business prior to Windows 10 version 1903 (May 2019 Update).*

<a href="" id="biometrics-enableESSwithSupportedPeripherals--only-for---device-vendor-msft-"></a>**Biometrics/EnableESSwithSupportedPeripherals** (only for ./Device/Vendor/MSFT)  

> If this policy is enabled, Windows Hello authentication using peripheral biometric sensors will be blocked. Any non-authentication operational functionalities such as camera usage (video calls, camera etc.) will be unaffected.

Reviewer: ```suggestion
If this policy is enabled, Windows Hello authentication using peripheral biometric sensors will be blocked. Any non-authentication operational functionalities such as camera usage (for instance, video calls and the camera) will be unaffected.
```

> If you enable this policy then it can have following possible values:

Reviewer: ```suggestion
If you enable this policy it can have the following possible values:
```

> 0 - Enhanced Sign-in Security Disabled (not recommended)

Reviewer: ```suggestion
**0 - Enhanced Sign-in Security Disabled** (not recommended)
```

Enhanced sign-in security will be disabled on all systems, enabling the use of peripheral biometric authentication. If this policy value is set to 0 after users have enrolled in ESS biometrics, users will be prompted to reset their PIN. They will lose all their existing biometric enrollments. To use biometrics they will have to enroll again. 

> 1 - Enhanced Sign-in Security Enabled (default and recommended for highest security)

Reviewer: ```suggestion
**1 - Enhanced Sign-in Security Enabled** (default and recommended for highest security)
```

Enhanced sign-in security will be enabled on systems with capable software and hardware, following the existing default behavior in Windows. Authentication operations of any biometric device that Enhanced Sign-in Security does not support, including that of peripheral devices, will be blocked and not available for Windows Hello.

> If you disable or do not configure this policy then Enhanced Sign-in Security is preferred on the device. The behavior will be same as enabling the policy and setting value to 1.

Reviewer: ```suggestion
If you disable or do not configure this policy, Enhanced Sign-in Security is preferred on the device. The behavior will be the same as enabling the policy and setting the value to 1.
```
